Automattic/jetpack
# Jetpack menu: only show to contributors and above.

## Proposed changes:

The menu isn't that useful to folks who cannot write posts on the site.

### Other information:


## Jetpack product discussion

* 

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site that's connected to WordPress.com.
* Go to Users > Add New and add a new user with a subscriber role.
* In a new browser, log in to your site with that user
    * You should not see any Jetpack menu.
* Now edit that user again and make it a contributor role.
* Back in the other browser, refresh and the Jetpack menu should appear.
